pkg/importer/http-datasource.go

```diff
@@ -130,20 +130,13 @@ func (hs *HTTPDataSource) Info() (ProcessingPhase, error) {
 	if hs.contentType == cdiv1.DataVolumeArchive {
 		return ProcessingPhaseTransferDataDir, nil
 	}
-	if hs.readers.Convert {
-		if hs.brokenForQemuImg || hs.readers.Archived || hs.customCA != "" {
-			return ProcessingPhaseTransferScratch, nil
-		}
-	} else {
-		if hs.readers.Archived || hs.customCA != "" {
-			return ProcessingPhaseTransferDataFile, nil
-		}
-	}
-	hs.url, _ = url.Parse(fmt.Sprintf("nbd+unix:///?socket=%s", nbdkitSocket))
-	if err = hs.n.StartNbdkit(hs.endpoint.String()); err != nil {
-		return ProcessingPhaseError, err
+	if !hs.readers.Convert && (hs.readers.Archived || hs.customCA != "") {
+		return ProcessingPhaseTransferDataFile, nil
```

Comment:
I think we can technically get rid `&& (hs.readers.Archived || hs.customCA != "")`?

No need to write a raw/iso to scratch, right?

UNLESS we want to truly respect `preallocation` settings.  In which case, we should just always `return ProcessingPhaseTransferScratch, nil` for kubevirt content type